ftrip.io.catalog-service/ftrip.io.catalog-service/Accommodations/UseCases/UpdateAccommodation/UpdateAccommodationRequest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using ftrip.io.catalog_service.Accommodations.Domain;
+using MediatR;
+using System;
+using System.Collections.Generic;
+
+namespace ftrip.io.catalog_service.Accommodations.UseCases.UpdateAccommodation
+{
+    public abstract class PartialAccommodationUpdateRequest: IRequest<Accommodation>
+    {
+        public Guid Id { get; set; }
```

Comment:
Place [JsonIgnore] on this one because we are getting it from the path parameters. If you open Swagger you can probably see it inside the body which we don't want to 😄.